obstacle_range: 2
raytrace_range: 3

footprint: [[-0.37, -0.17], [-0.37, 0.17], [0.2, 0.17], [0.2, -0.17]]

inflation_radius: 0.15
cost_scaling_factor: 25

map_type: costmap
observation_sources: scan
scan: {sensor_frame: base_scan, data_type: LaserScan, topic: scan, marking: true, clearing: true}